### PR TITLE
added presetations html

### PR DESCRIPTION
**Pull request by itsashishbharti**

### CODE REVIEW

**Review by amezng — CHANGES_REQUESTED** [review]
make the changes

**Comment by amezng** [comment]
@itsashishbharti fix the spelling in the title of the issue

File: index.html [hunk 1]
```diff
@@ -6,5 +6,31 @@
 </head>
 
 <body>
+	  <section id="Presentation">
+       		 <h2>Presentations</h2>
```
[the remark this answers is not in the document]
Comment (itsashishbharti) [reply index.html:10]: ok sir

**Comment by amezng** [comment]
this PR has been rejected due to incorrect commit messages. Use meaningful commit messages

**Comment by amezng** [comment]
this PR has been rejected due to incorrect commit messages. Use meaningful commit messages

**Pull request closed without merging**